File: app/src/at-vinhnguyen/javacore/OOP/Circle.java

package OOP;

public class Circle extends Shape{
    
    private float r;
    private final float PI = 3.14f;
    public Circle(float r) {
        this.r = r;
    }
    
    @Override
    float Acreage() {
        return PI*r*r;
    }

    @Override
    float Perimeter() {
        return r*2*PI;
    }

    @Override
    void printNameOfTheShape() {
        System.out.println("Circle:");
    }
}

File: app/src/at-vinhnguyen/java/asiantech/internship/summer/kotlincore/oop/Circle.kt
package asiantech.internship.summer.kotlincore.oop

class Circle(private val r: Float) : Shape() {
    private val PI = 3.14f

    override fun Acreage(): Float {
        return PI * r * r
    }

    override fun Perimeter(): Float {
        return r * 2f * PI
    }

    override fun printNameOfTheShape() {
        println("Circle:")
    }
}
